streams/streams-scala/src/test/scala/org/apache/kafka/streams/scala/StreamToTableJoinScalaIntegrationTestImplicitSerdes.scala

```diff
@@ -122,7 +120,7 @@ class StreamToTableJoinScalaIntegrationTestImplicitSerdes extends StreamToTableJ
 
     // Compute the total per region by summing the individual click counts per region.
     val clicksPerRegion: KTableJ[String, JLong] = clicksByRegion
-      .groupByKey(Serialized.`with`(Serdes.String, Serdes.JavaLong))
+      .groupByKey(Grouped.`with`[String, JLong](Serdes.String, Serdes.JavaLong))
```

Comment:
I don't think you should need those type parameters there. It'd be nice to verify that we don't need them, actually.

Reply:
This was put in by another comment.